README.md

```diff
@@ -1,2 +1,23 @@
 # My-Portfolio
+
 Portfolio
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+**This is a my personal portfolio website.**
```

Comment:
- [ ] Hello as requested by the [previous reviewer](https://github.com/asnganje/My-Portfolio/pull/1#discussion_r911264692), kindly remove this line here as it does not make your documentation to be professional. For an idea of how a professional looking README.md file looks like kindly have a look at [this example](https://github.com/betses/Portfolio/blob/main/README.md).